ouroboros-network-framework/src/Ouroboros/Network/Snocket.hs

```diff
@@ -280,7 +280,7 @@ type LocalHandle = Socket
 #endif
 
 -- | System dependent LocalSnocket type
-newtype LocalSocket  = LocalSocket { getLocalHandle :: LocalHandle }
+newtype LocalSocket  = LocalSocket { getLocalHandle :: LocalHandle } deriving (Eq, Show)
```

Comment:
You shouldn't need those instance, but adding them is a good idea.  I'll add this to #2772.

Reply:
I pushed this to #2772.  I used `deriving Show via Quiet LocalSocket` which will give us slightly nicer `Show` instance.